tests/test_financial.py

```diff
@@ -90,13 +93,167 @@ def test_decimal_with_when(self):
 
 
 class TestPV:
```

Comment:
fwiw, I would remove the test classes and rely on function naming convention (ex: `test_pv*`). Out of scope.

Reply:
I agree on principle, however having them in classes makes it easier to run groups of tests for each function.